google/cloud/forseti/common/util/parser.py

```diff
@@ -59,17 +59,17 @@ def format_timestamp(timestamp_str, datetime_formatter):
         str: The formatted, stringified timestamp.
     """
     try:
-        if '"' in timestamp_str or "'" in timestamp_str:
+        if '\'' in timestamp_str or '\'' in timestamp_str:
```

Comment:
I think the logic here needs to be updated to `if '"' in timestamp_str or '\'' in timestamp_str` instead

Reply:
done